packages/contracts-bedrock/scripts/getting-started/config-vars-celo.sh

```diff
@@ -65,10 +74,12 @@ config=$(cat << EOL
   "l1FeeVaultWithdrawalNetwork": 0,
   "sequencerFeeVaultWithdrawalNetwork": 0,
 
-  "gasPriceOracleOverhead": 2100,
+  "gasPriceOracleOverhead": 0,
   "gasPriceOracleScalar": 1000000,
 
-  "enableGovernance": true,
+  "deployCeloContracts": true,
```

Comment:
Is this to use together with migrated state? Then it should default to `false`.

Reply:
Good to know, I'll add as a `.envrc` variable then